### ISSUE TITLE
caffe to caffe2 translation error

### ISSUE THREAD

**Issue by karlybirdkarly** [facebookarchive/caffe2#1373]
**When I tried to convert caffe to caffe2 with the script** $ python -m caffe2.python.caffe_translator Desktop/deploy.prototxt Desktop/caffeModel.caffemodel  **it gives me:**
WARNING:root:This caffe2 python run does not have GPU support. Will run in CPU only mode.
WARNING:root:Debug message: No module named caffe2_pybind11_state_gpu
Traceback (most recent call last):
  File "/usr/local/anaconda3/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/anaconda3/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/local/caffe2/python/caffe_translator.py", line 920, in <module>
    open(input_proto, 'r').read(), caffenet
  File "/usr/local/anaconda3/lib/python2.7/site-packages/google/protobuf/text_format.py", line 525, in Merge
    descriptor_pool=descriptor_pool)
  File "/usr/local/anaconda3/lib/python2.7/site-packages/google/protobuf/text_format.py", line 579, in MergeLines
    return parser.MergeLines(lines, message)
  File "/usr/local/anaconda3/lib/python2.7/site-packages/google/protobuf/text_format.py", line 612, in MergeLines
    self._ParseOrMerge(lines, message)
  File "/usr/local/anaconda3/lib/python2.7/site-packages/google/protobuf/text_format.py", line 627, in _ParseOrMerge
    self._MergeField(tokenizer, message)
  File "/usr/local/anaconda3/lib/python2.7/site-packages/google/protobuf/text_format.py", line 727, in _MergeField
    merger(tokenizer, message, field)
  File "/usr/local/anaconda3/lib/python2.7/site-packages/google/protobuf/text_format.py", line 815, in _MergeMessageField
    self._MergeField(tokenizer, sub_message)
  File "/usr/local/anaconda3/lib/python2.7/site-packages/google/protobuf/text_format.py", line 695, in _MergeField
    (message_descriptor.full_name, name))
google.protobuf.text_format.ParseError: 1007:1 : Message type "caffe.LayerParameter" has no field named "layer".

So basically I have two types of issues, one with runpy and caffe_translator, and another is with protobuf. 

Protobuf was installed with $ conda install protobuf=3.1.0

$ which python
/usr/local/anaconda3/bin/python

$ python --version
Python 2.7.14 :: Anaconda custom (x86_64)

Any help, hint is appreciated! Thanks guys!
